core/remote/src/main/kotlin/team/ppac/remote/interceptor/ErrorInterceptor.kt

```diff
@@ -0,0 +1,33 @@
+package team.ppac.remote.interceptor
+
+import kotlinx.serialization.json.Json
+import okhttp3.Interceptor
+import okhttp3.Response
+import team.ppac.errorhandling.FarmemeException
+import team.ppac.errorhandling.parseWithNetworkError
+import javax.inject.Inject
+
+class ErrorInterceptor @Inject constructor(
+    private val json: Json,
+) : Interceptor {
+    override fun intercept(chain: Interceptor.Chain): Response {
+        try {
+            val response = chain.proceed(chain.request())
+            if(response.isSuccessful){
+                return response
+            }
+
+            //에러가 발생 하는 경우
+            response.body?.let {
+                val errorBody = json.decodeFromString<team.ppac.remote.model.response.Response<*>>(it.string())
```

Comment:
요거 뭐라고 부르더라..?? 대충 아래처럼 하는거 ㅇㅇ
```suggestion
import team.ppac.remote.model.response.Response as FMResponse
val errorBody = json.decodeFromString<FMResponse<*>>(it.string())
```

Reply:
import alias? 요거 적용해서 해볼게~